app-starter/WguAppTemplate.vue

```diff
@@ -3,7 +3,7 @@
 
```

Comment:
a cosmetic remark:
the map controls like basemap switcher or zoom buttons should have the primary color as well - as least we had this for our previous projects. See Screenshots:

Basemap Switcher:
![image](https://user-images.githubusercontent.com/15704517/140069874-8bbab525-0547-48b8-a030-de962db01790.png)

Zoom Buttons:
![image](https://user-images.githubusercontent.com/15704517/140069936-b70edace-0512-492f-9207-0c0aa167159a.png)

Reply:
Material design specifies that any floating button should take an accent color, so that they are distinguished from the other UI elements (in this case we use the secondary color).

In the end, both the primary and secondary can be set to the same colors, which is the effect that you are describing.

Reply:
alright :+1:

Reply:
In the latest commit, the default primary and secondary colors are the same tone of red :+1: